### ISSUE TITLE
Clarification of selectedSegmentIds

### ISSUE THREAD

**Issue by K-Meech** [mobie/mobie.github.io#61]
It would be useful to add a bit of clarification as to what a 'segment id' is here: https://github.com/mobie/mobie.github.io/blob/master/schema/view.schema.json#L197 

@constantinpape Am I right that it is "sourceName;timePoint;label_id";, like here: https://github.com/mobie/platybrowser-datasets/blob/spec-v2/data/1.0.1/dataset.json#L2927? (I need to update the bookmark saving to reflect this too)

**Comment by constantinpape** [comment]
> [](https://github.com/mobie/mobie.github.io/blob/master/schema/view.schema.json#L197)
> 
> @constantinpape Am I right that it is "sourceName;timePoint;label_id";, like here: https://github.com/mobie/platybrowser-datasets/blob/spec-v2/data/1.0.1/dataset.json#L2927? (I need to update the bookmark saving to reflect this too)

Yes, that's correct. Feel free to expand the description string to describe it better.

**Comment by K-Meech** [comment]
@constantinpape - just wanted to check, what's the format for the ```selectedAnnotationIds``` in ```sourceAnnotationDisplay```? https://github.com/mobie/mobie.github.io/blob/master/schema/view.schema.json#L286 Is it sourceName;annotationId? Thanks!

**Comment by constantinpape** [comment]
@K-Meech it's `timePoint;annotationId` (which should be added as documentation...)

**Comment by K-Meech** [comment]
Ok great! I can add lines to the docs for both of these :)
